charts/etos/charts/etos-api/templates/deployment.yaml

```diff
@@ -16,25 +16,47 @@ spec:
         app: {{ include "etos-api.name" . }}
         db: redis
     spec:
+      {{- if .Values.global.filebeatSidecar.enabled }}
+      volumes:
+      - name: shared-logs
+        emptyDir: {}
+      - name: filebeat-config
+        configMap:
+          name: sidecar-config
+      {{- end }}
       containers:
-      - name: {{ include "etos-api.name" . }}
-        {{- include "etos.containerImage" . | indent 8 }}
-        envFrom:
-        - configMapRef:
-            name: etos
-        env:
-          - name: RABBITMQ_PASSWORD
-            valueFrom:
-              secretKeyRef:
-                name: rabbitmq
-                key: password
-          - name: RABBITMQ_USERNAME
-            valueFrom:
-              secretKeyRef:
-                name: rabbitmq
-                key: username
-        ports:
-        - containerPort: 80
+        - name: {{ include "etos-api.name" . }}
```

Comment:
Indentation is wrong.

Reply:
Indentation is correct in the generated yaml. If I indent, I cannot deploy.